keyboards/2x5keypad/rules.mk

```diff
@@ -0,0 +1,56 @@
+# MCU name
+MCU = atmega32u4
```

Comment:
Actually, everything between this and bootmagic could be removed, as we've recently implemented defaults for many of the MCUs. 

In fact, you could just set this entire file to be: 

```
MCU = atmega32u4
BOOTLOADER = atmel-dfu

BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes	# Mouse keys
EXTRAKEY_ENABLE = yes	# Audio control and System control
CONSOLE_ENABLE= no	# Console for debug
COMMAND_ENABLE = no    # Commands for debug and configuration
SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
NKRO_ENABLE = yes	# USB Nkey Rollover -
BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
AUDIO_ENABLE = no
RGBLIGHT_ENABLE = no
```